docusaurus/docs/iOS/swiftui/components/composer-commands.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Message Composer Commands
+---
+
+## Composer Commands Overview
+
+The SwiftUI SDK has support for several types of commands in the composer. For example, when a user types "@" in the input field, a list of users that can be mentioned will be displayed. Additionally, the composer supports instant commands ("/"), similar to Slack. For example, you can share a giphy, by typing the "/giphy" command, or mute/unmute users. All the symbols and text for the commands are configurable, so you can use different displaying information for them. Additionally, you can create your own commands, and define their rules and handling. For example, you can create a "/pay" command, to send money to a user in the chat.
```

Comment:
There's a bit too much repetition in this sentence 🤔 The "For example" and "Additionally" are a bit abused. I guess we can mention all supported commands in one sentence, then explain how to use one of them, and then explain custom commands. Maybe @adamrushy can help here 🙂